app/controllers/users/circuitverse_controller.rb

```diff
@@ -45,11 +47,26 @@ def groups
 
     def profile_params
       params.require(:user).permit(:name, :profile_picture, :country, :educational_institute,
-                                   :subscribed, :locale, :remove_picture)
+                                   :subscribed, :locale, :remove_picture, :avatar)
     end
 
     def set_user
       @profile = current_user
       @user = User.find(params[:id])
     end
+
+    def attach_avatar
+      return if no_attachment?
+
+      pfp_present = params[:user][:profile_picture].present? && @profile.profile_picture.present?
```

Comment:
```suggestion
      profile_picture_present = params[:user][:profile_picture].present? && @profile.profile_picture.present?
```

Reply:
Line too long.. 😅 Changed to `pic_exists`